phf/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "phf"
 authors = ["Steven Fackler <sfackler@gmail.com>"]
-version = "0.7.24"
+version = "0.7.25"
```

Comment:
I feel like it's not really in the PR author's purview to edit the crate version number. I think the next release is going to be 0.8.0 anyway.